instruqt/proxy/01-proxy-config/setup-isolated-host

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -euxo pipefail
+
+echo "Running challenge 01 setup script - Started.."
+epoc_start=`date +%s`
+
+# drop egress (doesn't work with this in track_scripts setup stage as breaks instruqt setup)
+iptables -A OUTPUT -p tcp -m tcp -j DROP
+iptables -A OUTPUT -p icmp -j DROP
+
+# clear some daily package management tasks that cause dpkg lock when installing kurl
+systemctl stop apt-daily.timer
+systemctl disable apt-daily.timer
+systemctl mask apt-daily.service
+systemctl daemon-reload
+mv /usr/lib/apt/apt.systemd.daily /usr/lib/apt/apt.systemd.daily.DISABLED
+rm -f /etc/cron.daily/dpkg 
+rm -f /etc/apt/apt.conf.d/10periodic
+
+# bashrc
+echo "alias l='ls -latFrh'" >> /root/.bashrc
```

Comment:
I don't see this alias used anywhere in the lab, does it need to be added?

Reply:
Everyone has their favourite alias when working on things, I should have removed it before the pull request as not needed now